beefy-gadget/src/worker.rs

```diff
@@ -214,10 +239,10 @@ where
 
 		if self.should_vote_on(*notification.header.number()) {
 			let authority_id = if let Some(id) = self.key_store.authority_id(self.rounds.validators().as_slice()) {
-				trace!(target: "beefy", "🥩 Local authority id: {:?}", id);
+				debug!(target: "beefy", "🥩 Local authority id: {:?}", id);
 				id
 			} else {
-				trace!(target: "beefy", "🥩 Missing validator id - can't vote for: {:?}", notification.header.hash());
+				warn!(target: "beefy", "🥩 Missing validator id - can't vote for: {:?}", notification.header.hash());
```

Comment:
We need to tone this down, it's going to be triggered on every non-validator node and there is nothing the user can do to fix this.